mopidy/core/actor.py

```diff
@@ -136,6 +140,103 @@ def tags_changed(self, tags):
             self.playback._stream_title = title
             CoreListener.send('stream_title_changed', title=title)
 
+    def setup(self):
```

Comment:
Please add a docstring stating that this is for internal startup, and should not be called from frontends. Also does this show up in the JSON-RPC?